epan/dissectors/packet-ssh.c

```diff
@@ -1166,11 +1208,18 @@ if (offset == 0) {
 		g_print("WSDUMP KEY_DUMP_0 %s\n", chars1);
     // gint captured_length = tvb_captured_length(tvb);
     g_print("Offset %i\n", offset);
+   
     // if (offset == 0 && captured_length >= 4) {
     guchar* packet_len = (guchar*) tvb_get_ptr(tvb, offset, 4);
     guint32 payload_len = 0;
-    openssh_chacha20_poly1305_decrypt_len(global_data->key0, global_data->current_seqnr, packet_len, &payload_len);
-     offset += 4;
+        
+    ssh_packet_private_data_t* packet_data = ssh_get_packet_data(pinfo, FALSE, 0);
+    g_print("Found seqnr for len %u\n", packet_data->seqnr);
+       g_print("Found lens a %u and b %u\n", len, len_real);
+        g_print("Is sent: %s\n", packet_data->is_sent ? "yes" : "no");
```

Comment:
Tip: you can use something like `proto_tree_add_debug_text(tree, "Is sent: %s", packet_data->is_sent ? "yes" : "no");` for debugging. This will be visible in the tree and stderr.

Reply:
Working on removing all print statements!